Namco/position_comparison.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+import os
+
+
+class PositionComparison:
+    def check_collision(self) -> [bool,bool]:
+        """
+        Checks whether Pacman picked up a dot and increases the score accordingly
+        :return: wall collision flag (bool), dot collision flag (bool)
+        """
+        for dot in self.dots:
```

Comment:
A pontok és a falak nem koordinátával vannak mostmár tárolva, hanem a map tömbben. Szóval a collision check-et úgy kellene csinálni, hogy 
if map[x][y] == '-' or map[x][y] == '|':
    wall_collision = true
elif map[x][y] == '0':
    dot_collision = true

Reply:
Megcsináltam igy is, dotok ugye már x-el vannak jelölve ha jól néztem.